### PR TITLE
Align invite members button on the team page

### PR DESCRIPTION
**Pull request by gtsiolis**
Fix https://github.com/gitpod-io/gitpod/issues/5092.

| BEFORE | AFTER |
|-|-|
| button-before | button-after |

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **alextugarev** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @alextugarev` in a comment when ready.

Associated issue: *#5092*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
@svenefftinge could you take a look at this one-line change since you were involved in https://github.com/gitpod-io/gitpod/pull/4868? 🏀

**Comment by jankeromnes** [comment]
I've added a Team & a Project to test this 😁 if you want to join that team: https://gt-align-invite-members-button.staging.gitpod-dev.com/teams/join?inviteId=ecb8c1b3-00ec-494b-b64f-dd5db4a1f0b0

**Review by jankeromnes — APPROVED** [review]
DOES. THE. JOB. 🚀 🚀 🚀

/approve

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 74766f5da2187916ff925054d6b4b34b04840c9a</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5093#pullrequestreview-724343291" title="Approved">jankeromnes</a>*

Associated issue: *#5092*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by gtsiolis** [comment]
Thanks @jankeromnes! 🧑‍🚀

**Pull request merged**
